### DISCUSSION TITLE
Only downloading 1 episode when adding a TV show on watchlist

### DISCUSSION THREAD

**Discussion by dinkohumd** [itsToggle/plex_debrid#600] [Q&A]
When I add a TV show on the watchlist, it only adds single episode. It seems like it tries to download the others, but fails...
<img width="969" alt="Screenshot 2023-12-09 at 23 10 18" src="https://github.com/itsToggle/plex_debrid/assets/153403521/f30578fc-928d-49c7-a9f6-0f0f00429a2b">

**Comment by GuillermoMantegna** [comment]
hi! could you resolve this issue? it happens to me with new seassons of some shows.

**Comment by GuillermoMantegna** [comment]
[This](https://github.com/itsToggle/plex_debrid/issues/578#issuecomment-1841483489) temporary fix solved the issue for me.

**Reply by dinkohumd** [reply]
Ah, thanks for the temp fix.  Appreciate it.  It's been so frustrating for me.
